src/commands/serve.ts

```diff
@@ -30,9 +30,12 @@ export class ServeCommand implements Command {
 
   args = polyserveArgs;
 
-  run(options: CommandOptions, config: ProjectConfig): Promise<any> {
+  async run(options: CommandOptions, config: ProjectConfig): Promise<any> {
     // Defer dependency loading until this specific command is run
     const polyserve = require('polyserve');
+    const startServers = polyserve.startServers;
+    const getServerUrls = polyserve.getServerUrls;
+    const url = require('url');
```

Comment:
fwiw `url` is probably fine to import at the top of the file, it's small and I'm sure it's already been imported somewhere else at this point. Fine if you want to keep it here though.